FPWD/2020-06-18/index.html

```diff
@@ -2902,12 +2902,24 @@ <h2 id="x6-did-methods"><bdi class="secno">6. </bdi>DID Methods<a class="self-li
 
 
 
-<section id="references" class="appendix"><h2 id="a-references"><bdi class="secno">A. </bdi>References<a class="self-link" aria-label="§" href="#references"></a></h2><section id="normative-references">
+<section id="references" class="appendix"><h2 id="a-references"><bdi class="secno">A. </bdi>References<a class="self-link" aria-label="§" href="#references"></a></h2>
```

Comment:
The 2020-06-18 fpwd is a frozen document, this file shouldn't be changed.

Reply:
@jonnycrunch suggest you resolve this.

Reply:
I think I update the correct version and undid the changes to head index,  but please advise.

Reply:
The only file in "files changed" should be `index.html`

https://github.com/w3c/did-spec-registries/pull/138/files

It appear you are still committing changes to FPWD.

Reply:
ok, hard reset and hopefully resolved my error.  Thanks!